cmd/get/matchable_cluster_resource_attribute.go

```diff
@@ -11,12 +11,12 @@ import (
 )
 
 const (
-	clusterResourceAttributesShort = "Gets matchable resources of cluster resource attributes"
+	clusterResourceAttributesShort = "Gets matchable resources of cluster resource attributes."
 	clusterResourceAttributesLong  = `
-Retrieves cluster resource attributes for given project and domain combination or additionally with workflow name.
+Retrieves cluster resource attributes for the given project and domain, combination or additionally with workflow name.
```

Comment:
@evalsocket, can we remove this as the explanation is already present in the 18th line?